doc/userdoc/contribute/coding_guidelines_cpp.rst

```diff
@@ -62,12 +58,12 @@ Ubuntu see `here <http://llvm.org/apt/>`_:
 
    # install clang-format with:
    sudo apt-get update
-   sudo apt-get install libstdc++6 libllvm3.6 clang-format-3.6
+   sudo apt-get install clang-format-9
 
 OS X:
 
-* Installing ``clang-format-3.6`` from the `pre-built
-  binaries <http://llvm.org/releases/3.6.2/clang+llvm-3.6.2-x86_64-apple-darwin.tar.xz>`_.
+* Installing ``clang-format`` from the `pre-built binaries
+  <https://github.com/llvm/llvm-project/releases/download/llvmorg-9.0.1/clang+llvm-9.0.1-x86_64-apple-darwin.tar.xz`_.
```

Comment:
Could you extend this point in the following direction (don't have the rst syntax at my fingertips):
1. Download and unpack `pre-built Clang+LLVM-9.0.1 binaries <...>`
2. Copy `bin/clang-format` from the archive to `clang-format-9` in a location where it will be found, e.g., `/usr/local/bin`.

Reply:
Fixed in 2c48e18 together with the removal of the detailed installation instructions for Ubuntu.